src/main/kotlin/no/nav/familie/pdf/pdf/domain/FeltMap.kt

```diff
@@ -0,0 +1,18 @@
+package no.nav.familie.pdf.pdf.domain
+
+import jakarta.validation.constraints.NotNull
+
+data class FeltMap(
+    @field:NotNull(message = "Label kan ikke være null")
```

Comment:
Dette trengs ikke i Kotlin - alt er "not null" by default, med mindre du legger til `?` 

Nullable: `var label: String?`
Ikke nullable: `var label: String`

Reply:
Da fjerner jeg 👼🏼 !